scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -251,6 +252,11 @@ def generate_subset(self, subset, sets):
         else:
             self.instances = OrderedDict(sorted(self.instances.items()))
 
+        if self.options.shuffle_tests:
+            temp_list = list(self.instances.items())
+            random.shuffle(temp_list)
```

Comment:
can we add a sed here, and export it? so that we can always have a way to reproduce it? otherwise, we may have problem to reproduce such issues.

Reply:
+ rebased.